vim.yaml

```diff
@@ -5,6 +5,9 @@ package:
   description: "Improved vi-style text editor"
   copyright:
     - license: Vim
+    - license-path: src/libvterm/LICENSE
+      license: Apache-2.0
+      detection-override: MIT
```

Comment:
what's this?! =)

Reply:
Accidental push from my license linting demo! ;) Removed it with subsequent commit. I'll squash all commits later.